lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
> We probably need better tests

What tests exactly? Some more tests for this method in `spec/lib/account_deleter_spec.rb`?